app/uk/gov/hmrc/plasticpackagingtax/returns/config/AppConfig.scala

```diff
@@ -65,6 +65,7 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
 
   lazy val pptRegistrationInfoUrl: String = config.get[String]("urls.pptRegistrationsInfoLink")
   lazy val pptGuidanceUrl: String         = config.get[String]("urls.pptGuidanceLink")
+  lazy val pptCompleteReturnUrl: String   = config.get[String]("urls.pptCompleteReturnLink")
```

Comment:
Might be worth making it clear that this is a guidance/info link? It confused me until I read on.

Reply:
Good idea - changed to `pptCompleteReturnGuidanceUrl`